src/components/status-image/status-image.css

```
@@ -1,4 +1,21 @@
 .StatusImage {
+    
     border: 1px solid white;
-    background-color: var(--color-busy);
+    height: 423px;
+    width: 414px;
+}
+
+img {
```

Only apply this when it is inside the componenet. Use class as selector. Add a class to the img in the html code to know if the image is for the background or for the person.
```suggestion
.StatusImage img.background {
```